system/Session/Handlers/MemcachedHandler.php

```diff
@@ -93,12 +93,12 @@ public function open($path, $name): bool
         }
 
         if (
-            ! preg_match_all(
+            in_array(preg_match_all(
```

Comment:
the in_array looks overly complicated. can we do just int cast then !== 0?

Reply:
should be `=== 0`, since it negation?

Reply:
Yes, you're right - it should be `=== 0`

Reply:
ok, updated https://github.com/codeigniter4/CodeIgniter4/pull/9339/commits/aa6f5eedeeda65aeab23f3d36bd8fa074ab34b0a